Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,26 +1,165 @@
 import React from 'react';
-import RightButtonsContainer from './components/ButtonComponents/RightButtonsContainer';
-import LeftButtonsContainer from './components/ButtonComponents/LeftButtonsContainer';
+import CalcButtonsContainer from './components/ButtonComponents/CalcButtonsContainer';
 import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
 
+function translateLabel(label){
+  if(label==='÷'){
+    return '/';
+  }else if(label==='×'){
+    return '*';
+  }else{
+    return label;
+  }
+}
+
+function basicMaths(a,b,operand){
+
+  if(operand==="+"){
+    return a+b;
+  }else if(operand==="-"){
+    return a-b;
+  }else if(operand==="/"){
+    return a/b;
+  }else if(operand==="*"){
+    return a*b;
+  }
+
+}
+
+function isCharacterANumber(character){
+  if(character==="+"){
+    return false;
+  }else if(character==="-"){
+    return false;
+  }else if(character==="/"){
+    return false;
+  }else if(character==="*"){
+    return false;
+  }else if(character==="="){
+    return false;
+  }else{
```

Comment:
Consider shortening this down by using something like the string [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) method:
```javascript
if('+-/*='.includes(character)) return false;
else return true;
```
or better yet:
```javascript
return !('+-/*='.includes(character));
```